a3p-integration/proposals/z:acceptance/vaults.test.js

```diff
@@ -0,0 +1,201 @@
+import test from 'ava';
+import {
+  agops,
+  bankSend,
+  getUser,
+  openVault,
+  adjustVault,
+  closeVault,
+  getISTBalance,
+  queryVstorage,
+  ATOM_DENOM,
+  USER1ADDR,
+  waitForBlock,
+} from '@agoric/synthetic-chain';
+
+const getFormattedVaultData = vaultData => {
+  const { values } = JSON.parse(vaultData.value);
+  const { body } = JSON.parse(values[0]);
+  const cleanedBody = JSON.parse(body.slice(1));
+  return cleanedBody;
+};
+
+test.serial('attempt to open vaults under the minimum amount', async t => {
+  const activeVaultsBefore = await agops.vaults('list', '--from', USER1ADDR);
+  await bankSend(USER1ADDR, `20000000${ATOM_DENOM}`);
+
+  const mint = '3.0';
+  const collateral = '5.0';
+  await t.throwsAsync(() => openVault(USER1ADDR, mint, collateral));
```

Comment:
Thank you for pointing this out.

The error message triggered by `openVault` is considerably large (included bellow). 
Having this in mind, I thought that it would be better to assert that the error message contains the string:
`Error: Vault creation requires a minInitialDebt of {"brand":"[Alleged: IST brand]","value":"[5000000n]"}`

Do you agree with this approach?

<details>

<summary>error.message logs</summary>

Command failed with exit code 1: /usr/src/agoric-sdk/node_modules/.bin/agops perf satisfaction --from agoric1rwwley550k9mmk6uq6mm6z4udrg8kyuyvfszjk --executeOffer /tmp/agops.0Eb '--keyring-backend=test'

      {
        sharedOpts: { keyringBackend: 'test' },
        opts: {
          from: 'agoric1rwwley550k9mmk6uq6mm6z4udrg8kyuyvfszjk',
          executeOffer: '/tmp/agops.0Eb'
        }
      }
      Following :published.wallet.agoric1rwwley550k9mmk6uq6mm6z4udrg8kyuyvfszjk
      wallet update {
        currentAmount: {
          brand: Object [Alleged: SEVERED: Zoe Invitation brand] {},
          value: []
        },
        updated: 'balance'
      }
      wallet update {
        status: {
          error: 'Error: Vault creation requires a minInitialDebt of {"brand":"[Alleged: IST brand]","value":"[5000000n]"}',
          id: 'openVault-1727263883705',
          invitationSpec: {
            callPipe: [Array],
            instancePath: [Array],
            source: 'agoricContract'
          },
          proposal: { give: [Object], want: [Object] }
        },
        updated: 'offerStatus'
      }
      Error: Vault creation requires a minInitialDebt of {"brand":"[Alleged: IST brand]","value":"[5000000n]"}

      Executing  [
        '--node=http://0.0.0.0:26657',
        '--chain-id=agoriclocal',
        '--from=agoric1rwwley550k9mmk6uq6mm6z4udrg8kyuyvfszjk',
        'tx',
        'swingset',
        'wallet-action',
        '--allow-spend',
        '{"body":"#{\\"method\\":\\"executeOffer\\",\\"offer\\":{\\"id\\":\\"openVault-1727263883705\\",\\"invitationSpec\\":{\\"callPipe\\":[[\\"getCollateralManager\\",[\\"$0.Alleged: BoardRemoteATOM brand\\"]],[\\"makeVaultInvitation\\"]],\\"instancePath\\":[\\"VaultFactory\\"],\\"source\\":\\"agoricContract\\"},\\"proposal\\":{\\"give\\":{\\"Collateral\\":{\\"brand\\":\\"$0\\",\\"value\\":\\"+5000000\\"}},\\"want\\":{\\"Minted\\":{\\"brand\\":\\"$1.Alleged: BoardRemoteIST brand\\",\\"value\\":\\"+3000000\\"}}}}}","slots":["board05557","board0257"]}',
        '--keyring-backend=test',
        '--yes'
      ]
</details>